cdap-watchdog/src/main/java/co/cask/cdap/metrics/process/MigrationTableHelper.java

```diff
@@ -31,10 +31,16 @@
  */
 final class MigrationTableHelper {
 
+  private static volatile boolean stopping = false;
```

Comment:
I am not sure. Static volatile... might work. But returning null? Do the callers expect the null?